libs/ptg-ui-web-components/src/components/ptg-progress-bar/ptg-progress-bar.scss

```diff
@@ -0,0 +1,126 @@
+.progress-bar{
+    height : 25px;
+    width : 100%;
+    border-radius: 50px;
+    background-color: rgb(205, 220, 223) !important;
+}
+
+.progress-fill{
+    height : 98%;
+    border-radius: 50px;
+    color: white;
+    background-color: rgb(139, 201, 212);
+}
+
+// .progress-fill-gradient{
+//     height : 98%;
+//     border-radius: 50px;
+//     color: white;
+//     font-size: 20px;
+//     background-image: linear-gradient(to right, #020024 , #00d4ff);
+// }
+
+
+// .dynamic-progress-fill{
+//     height : 98%;
+//     border-radius: 30px;
+//     color: white;
+//     width: 0;
+//     background-color: rgb(139, 201, 212);
+//     // animation: move 7s linear;
+// }
```

Comment:
@Pranjal26101999 Remove all the commented code from the progress bar if it is of no use.